agent/command/deprecated.go

```diff
@@ -56,3 +57,15 @@ func (cc *shellTrack) Execute(ctx context.Context,
 	logger.Execution().Warning("shell.track is deprecated. Process tracking is now enabled by default.")
 	return nil
 }
+
+type manifestLoad struct{ base }
+
+func manifestLoadFactory() Command                                      { return &manifestLoad{} }
+func (c *manifestLoad) Name() string                                    { return evergreen.ManifestLoadCommandName }
+func (c *manifestLoad) ParseParams(params map[string]interface{}) error { return nil }
+func (c *manifestLoad) Execute(ctx context.Context,
+	comm client.Communicator, logger client.LoggerProducer, conf *internal.TaskConfig) error {
+
+	logger.Execution().Warning("manifest.load is deprecated. Manifest load is now called automatically in git.get_project.")
```

Comment:
Can we use manifestLoadCommandName in this warning?

Reply:
done